File: java/src/main/java/io/eventdriven/slimdownaggregates/slimmed/events/BookApprovedEvent.java
package io.eventdriven.slimdownaggregates.slimmed.events;

import io.eventdriven.slimdownaggregates.slimmed.core.IDomainEvent;
import io.eventdriven.slimdownaggregates.slimmed.entities.BookId;
import io.eventdriven.slimdownaggregates.slimmed.entities.CommitteeApproval;

public record BookApprovedEvent(BookId id, CommitteeApproval committeeApproval) implements IDomainEvent {
}

File: csharp/SlimDownYourAggregates.Tests/Slimmed/Events/BookApprovedEvent.cs
using SlimDownYourAggregates.Tests.Slimmed.Core;
using SlimDownYourAggregates.Tests.Slimmed.Entities;

namespace SlimDownYourAggregates.Tests.Slimmed.Events;

public record BookApprovedEvent(
    BookId BookId,
    CommitteeApproval CommitteeApproval
): IDomainEvent;
